ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/data/OfflineModelsRepository.kt

```diff
@@ -0,0 +1,12 @@
+package ai.nnstreamer.ml.inference.offloading.data
+
+import kotlinx.coroutines.flow.Flow
+
+
+class OfflineModelsRepository(private val modelDao: ModelDao) : ModelsRepository {
```

Comment:
It's just a question, do you have a plan to make other ModelsRepository class? (OnlineModelsRepository..?)

Reply:
Ah, the term 'Repository' is used here in different contexts. Please refer to this [article](https://developer.android.com/topic/architecture/data-layer/offline-first#reads).

Reply:
Oh, I thought using interface is for another class like online.
There is a lot of content in the article that I am seeing for the first time.
Thank you :)